DataFormats/Detectors/FIT/FV0/include/DataFormatsFV0/Digit.h

```diff
@@ -0,0 +1,112 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FV0_DIGIT_H
+#define ALICEO2_FV0_DIGIT_H
+
+#include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/TimeStamp.h"
+#include <iosfwd>
+#include "Rtypes.h"
+
+namespace o2
+{
+namespace fv0
+{
+
+struct ChannelData {
+  //Int_t ChId;       // channel Id
+  //Double_t CFDTime; // time in ns, 0 at lhc clk center
+  //Double_t QTCAmpl; // Amplitude in mips
+  //int numberOfParticles;
+  Int_t mPmtNumber;   // PhotoMultiplier number (0 to 47)
```

Comment:
Do you really need an int here (according to the comment on the range: not). If you change it to short and place the mChargeAdc right after it, the ``sizeof(Digit)`` will become 8 bits instead of 12.
Could you please eliminated the obsolete commented code? Also note, that the according to the coding conventions, we don't need to prepend the data members of the struct by ``m``. So, I would have this just as
````
struct ChannelData {
  short pmtNumber = 0;
  short chargeAdc = 0;
  float time = 0.f;
  ClassDefNV(ChannelData,1);
};
````

Reply:
This is done